Examples/ArcGISToolkitExamples/ARExample.swift

```diff
@@ -35,15 +84,62 @@ class ARExample: UIViewController {
             arView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
             ])
         
-        arView.sceneView.scene = makeStreetsScene()
-        arView.locationDataSource = AGSCLLocationDataSource()
+        // Create a toolbar and add it to the arView.
+        let toolbar = UIToolbar(frame: .zero)
+        arView.addSubview(toolbar)
+        toolbar.translatesAutoresizingMaskIntoConstraints = false
+        NSLayoutConstraint.activate([
+            toolbar.leadingAnchor.constraint(equalTo: arView.sceneView.leadingAnchor),
+            toolbar.trailingAnchor.constraint(equalTo: arView.sceneView.trailingAnchor),
+            toolbar.bottomAnchor.constraint(equalTo: arView.sceneView.attributionTopAnchor)
+            ])
+        
+        // Create a toolbar button to change the current scene.
+        let sceneItem = UIBarButtonItem(title: "Change Scene", style: .plain, target: self, action: #selector(changeScene(_:)))
+        
+        // Create a toolbar button to display the status.
+        let statusItem = UIBarButtonItem(title: "Status", style: .plain, target: self, action: #selector(showStatus(_:)))
+
+        toolbar.setItems([UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil),
+                          sceneItem,
+                          UIBarButtonItem(barButtonSystemItem: .flexibleSpace, target: nil, action: nil),
+                          statusItem], animated: false)
+        
+        // Add the status view and setup constraints.
+        if let controller = statusViewController {
+            view.addSubview(controller.view)
```

Comment:
You're adding the view controller's view as a subview, but you're not making the view controller a child. That works, but is effectively no different than if `controller` were just a view, not a view controller. I would suggest either making this a [container view controller](https://developer.apple.com/library/archive/featuredarticles/ViewControllerPGforiPhoneOS/ImplementingaContainerViewController.html#//apple_ref/doc/uid/TP40007457-CH11-SW12) by adding `controller` as a child view controller or making `ARStatusViewController` into `ARStatusView`.

Reply:
I am now calling `addChild(statusViewController)` from my parent view controller.  I'm still unclear on the reason or benefit.